packages/react-native-bridge/index.js

```diff
@@ -275,7 +275,10 @@ export function requestMediaEditor( mediaUrl, callback ) {
 	);
 }
 
-export function fetchRequest( path ) {
+export function fetchRequest( path, enableCaching = true ) {
+	if ( isAndroid ) {
+		return RNReactNativeGutenbergBridge.fetchRequest( path, enableCaching );
+	}
```

Comment:
Request caching is only available on Android.

Reply:
Good approach since the bridge code on iOS does not contain `enableCaching` here right? https://github.com/WordPress/gutenberg/blob/f549c3245b8de3e2c5636f4e5e41900e101a5797/packages/react-native-bridge/ios/RNReactNativeGutenbergBridge.swift#L240

Reply:
Yep, actually, I was surprised that we have the caching feature only on Android 🤔 .